EIPS/eip-draft_hfmeta_homestead.md

```diff
@@ -22,9 +22,7 @@ This specifies the changes included in the hard fork named Homestead.
   - Block >= 0 on future testnets
 - Included EIPs:
   - EIP 2 (Homestead Hard-fork Changes)
-  - EIP 6 (Renaming Suicide Opcode)
   - EIP 7 (DELEGATECALL)
-  - EIP 8 (devp2p Forward Compatibility Requirements for Homestead)
```

Comment:
Need to remove 6 and 8 from the `requires` field too. I'm not sure 8 is not required for the fork?